test/unit/lexer/update.ts

```diff
@@ -0,0 +1,13 @@
+import { Update } from "../../../src/lexer/update";
+import { Query, getQueryFromLine } from "../../../src/reader/reader";
+
+it('should tokenise update keyword', () => {
```

Comment:
Wheres the assertion in this?

Reply:
I should have added that it's a WIP. Still working on mocking it xD